app/server/index.js

```diff
@@ -58,19 +58,27 @@ app.use(compression());
 app.use(bodyParser.urlencoded());
 app.use(bodyParser.json());
 
-app.use("/css/leaflet.css", express.static(path.join(__dirname, '../../node_modules/leaflet/dist/leaflet.css')));
+app.use("/leaflet.css", express.static(path.join(__dirname, '../../node_modules/leaflet/dist/leaflet.css')));
 app.use("/socket.io.js", express.static(path.join(__dirname, '../../node_modules/socket.io/node_modules/socket.io-client/socket.io.js')));
-app.use("/css", express.static(path.join(__dirname, '../client/css')));
-app.use("/images", express.static(path.join(__dirname, '../client/images')));
+app.use("/Map", express.static(path.join(__dirname, '../clients/Map')));
+app.use("/Admin", express.static(path.join(__dirname, '../clients/Admin')));
+app.use("/_common", express.static(path.join(__dirname, '../clients/_common')));
 
 app.get('/', function(req, res){
-    res.sendFile(path.join(__dirname, '../client/index.html'));
+    res.sendFile(path.join(__dirname, '../clients/Map/index.html'));
 });
 
-app.get('/browserify-bundle.js', function(req, res){
-    res.sendFile(path.join(__dirname, '../client/browserify-bundle.js'));
+app.get('/Admin', function(req, res){
```

Comment:
It'd be better to have a separate server file for the admin. Among other things, it'd allow separate deployment and starting the admin in a separate domain (like `admin.6element.ants.builders`)